packages/test/test-end-to-end-tests/src/packageVersion.ts

```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluidframework/test-end-to-end-tests";
-export const pkgVersion = "0.53.0";
+export const pkgVersion = "0.54.0";
```

Comment:
Is this a left-over from not fully complete release bump?

Reply:
Yes. This was auto-updated on a local build and now updated in `main` in a post-release PR